components/scream/tests/uncoupled/rrtmgp/CMakeLists.txt

```diff
@@ -22,13 +23,29 @@ if (NOT ${SCREAM_BASELINES_ONLY})
           ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/extensions/cloud_optics
     )
 
+    ## Create free running rrtmgp stand alone test that runs from an initial condition file.
+    CreateUnitTest(
```

Comment:
Seems like a bit redundant. The existing test was already running through the AD. Even though it was just running a single time step, it was still loading initial conditions via the AD initialization, no?

Reply:
no this test is comparing against a baseline file and has the initial condition set by a rrtmgp file. Before now we didn't have an rrtmgp standalone file that could initialize and run multiple timesteps.

Reply:
Is there a point in keeping the other test then?